lib/omnibus/packager.rb

```diff
@@ -68,7 +68,15 @@ def for_current_system
         family = "solaris"
       end
       if klass = PLATFORM_PACKAGER_MAP[family]
-        klass.is_a?(Array) ? klass : [ klass ]
+        package_types = klass.is_a?(Array) ? klass : [ klass ]
+
+        if package_types.include?(APPX) &&
+          !Chef::Sugar::Constraints::Version.new(version).satisfies?('>= 6.2')
```

Comment:
I believe the following less-verbose check should work also:

```ruby
if package_types.include?(APPX) && platform_version.satisfies?('>= 6.2')
```

`platform_version` is part of Chef Sugar's DSL and is brought in by way of the `Omnibus::Sugarable` mixin:
https://github.com/chef/omnibus/blob/master/lib/omnibus/sugarable.rb#L29-L36
https://github.com/sethvargo/chef-sugar/blob/master/lib/chef/sugar/platform.rb#L276-L286